### PR TITLE
Added a setExtras() setter which overwrites the instantiation extras

### PR DESCRIPTION
**Pull request by yasenslavov**
Added a setExtras() setter which overwrites the instantiation extras and triggers a calendar re-render. Also added an entry in the README.md to reflect the new interface.

### CODE REVIEW

File: src/clndr.js [hunk 1]
```diff
@@ -1530,6 +1530,17 @@
     };
 
     /**
+     * Overwrites extras in the calendar and triggers a render.
+     */
+    Clndr.prototype.setExtras = function(extras){
```
Comment (mikegioia) [inline src/clndr.js]: Could you reformat: `= function (extras) {`

File: src/clndr.js [hunk 2]
```diff
@@ -1530,6 +1530,17 @@
...
+        this.options.extras = extras;
+
```
Comment (mikegioia) [inline src/clndr.js]: Remove new line

File: src/clndr.js [hunk 3]
```diff
@@ -1530,6 +1530,17 @@
...
+        this.render();
+
```
Comment (mikegioia) [inline src/clndr.js]: remove new line

**Comment by mikegioia** [comment]
@yasenslavov These are very minor changes but it honors the style guide :D

**Comment by yasenslavov** [comment]
Haha, minor indeed. I needed it for a project, so I thought why not PR it, people may need it as I did.

**Comment by yasenslavov** [comment]
Apologies, I will push the changes via my other account in a separate PR, I will be disabling this one, I really hope that's fine.

**Pull request closed without merging**